### DISCUSSION TITLE
File:/// type Uri not loading in compose

### DISCUSSION THREAD

**Discussion by cj1098** [coil-kt/coil#1247] [General]
I'm using 2.0.0-rc02 and trying to load this Uri `file:///data/user/0/com.app.name/cache/1257586825.pdf` but keep getting a blank white screen after loading. Is there something I'm missing? I thought File was supposed to be supported?

Here's my Composable
`AsyncImage(model = uri,
                    contentDescription = null,
                    modifier = Modifier
                        .requiredSize(99.dp, 132.dp)
                        .clickable {
                            onItemClicked(uri)
                        }, contentScale = ContentScale.Crop
                )`

**Comment by colinrtwhite** [comment]
File URIs are supported. I'd check out the exception emitted in `onError` to see where in the decode process the request fails.
